api_app/service_bus/deployment_status_update.py

```diff
@@ -18,217 +22,227 @@
 from resources import strings
 
 
-async def receive_message():
-    """
-    This method is an async generator which receives messages from service bus
-    and yields those messages. If the yielded function return True the message is
-    marked complete.
-    """
-    async with credentials.get_credential_async() as credential:
-        service_bus_client = ServiceBusClient(config.SERVICE_BUS_FULLY_QUALIFIED_NAMESPACE, credential)
+class DeploymentStatusUpdate(threading.Thread):
```

Comment:
Nit: DeploymentStatusUpdater (ending with r), as it's a class?

Reply:
harsh...but fair :)